include/crud-terms.php

```diff
@@ -189,16 +187,14 @@ protected function get_queried_languages( $taxonomies, $args ): array {
 	}
 
 	/**
-	 * Adds language dependent cache domain when querying terms.
-	 * Useful as the 'lang' parameter is not included in cache key by WordPress.
+	 * Adjusts query language for `_get_term_hierarchy()` and `WP_Query`.
 	 *
 	 * @since 1.3
 	 *
-	 * @param array    $args       WP_Term_Query arguments.
-	 * @param string[] $taxonomies Queried taxonomies.
+	 * @param array $args WP_Term_Query arguments.
 	 * @return array Modified arguments.
 	 */
-	public function get_terms_args( $args, $taxonomies ) {
+	public function get_terms_args( $args ) {
```

Comment:
I suggest we rename this method so it describes what it does.
Some could say it has been removed completely because of bad naming 😅